.gitignore

```diff
@@ -95,3 +95,5 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+.vscode/
```

Comment:
Dont add .vscode/ to gitignore. It is debug config. 

For debuging you should remove docker container with FreeCAD (save project before) and run startup script in debug mode - `bash run.bash -d`. After that you can use VSCODE debuger and investigate code variables.

For returning to regular mode remove container and run as usual `bash run.bash`

Reply:
One more tip about FC dev. Reload FC by `bash run.bash` or `bash run.bash -d` after changing of code because it does not load new code automatically.

Reply:
> Dont add .vscode/ to gitignore. It is debug config.

Done , I've updated .gitignore

Reply:
> One more tip about FC dev. Reload FC by `bash run.bash` or `bash run.bash -d` after changing of code because it does not load new code automatically.

Thanks for the tips